src/keypress.c

```diff
@@ -314,9 +313,9 @@ void typeStringDelayed(const char *str, const unsigned cpm)
 		}
 
 		unicodeTap(n);
-
-		if (mspc > 0) {
-			microsleep(mspc + (DEADBEEF_UNIFORM(0.0, 62.5)));
+    
+    if (mspc > 0) {
+			microsleep(mspc);
 		}
 	}
-}
+}
```

Comment:
Please use tabs... it's a C thing. Also leave the new line at the end of the file. Thank you